templates/demo.html

```diff
@@ -30,8 +51,8 @@
           app: "com.vectorunit.cobalt.canonical", // Given
         },
         screen: {
-          width: width && width < 1104 ? width : 1104,
-          height: height && height < 628 ? height : 628,
+          width: streamWidth,
+          height: streamHeight,
```

Comment:
I was thinking a bit more about this. Can't we just scale the video element instead of assigning a static video size we can't change once the video is active?

Reply:
@morphis it would be simpler to set a fixed size with the downside that users with smaller screens will be streaming pixels they don't need. If we're ok with this inefficiency, then I can remove the resizing.